demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/transport_tls_esp32.c

```diff
@@ -90,6 +90,20 @@ TlsTransportStatus_t TLS_Socket_Connect( NetworkContext_t * pNetworkContext,
     {
         esp_transport_ssl_set_cert_data_der( pNetworkContext->xTransport, ( const char * ) pNetworkCredentials->pucRootCa, pNetworkCredentials->xRootCaSize );
     }
+#if CONFIG_ESP_TLS_USE_SECURE_ELEMENT
+
+    esp_transport_ssl_use_secure_element( pNetworkContext->xTransport );
+
+    #ifndef CONFIG_ATECC608A_TNG
+        /* This is either a TrustFLEX or a TrustCUSTOM chip - generate (using atcacert_* API) or plug in 
```

Comment:
This is unclear. I'm not familiar with TrustFLEX vs TrustCUSTOM (I assume it's a discussion between using the factory provisioned ATECC certificates vs custom ones?)
Please clarify or add code on how one would add their certificate.

Reply:
> (I assume it's a discussion between using the factory provisioned ATECC certificates vs custom ones?)

You are correct - it is indeed about that.

> Please clarify or add code on how one would add their certificate.

Let me work on this and commit. Also, I noticed an error in my comment. The parts that need no action are TrustFLEX and TrustCUSTOM. Thank you for bringing this up.

Reply:
@CIPop I have corrected the comments as well as added code that plugs in the custom device certificate.

c405422edd81a669450fafc1b667cb69ddc5e584